components/core/src/Core.js

```diff
@@ -0,0 +1,296 @@
+			// Textareas should really only resize vertically so they don't break their (horizontal) containers.
+			resize: 'vertical',
+		},
+
+		fieldset: {
+			// Browsers set a default `min-width: min-content;` on fieldsets,
+			// unlike e.g. `<div>`s, which have `min-width: 0;` by default.
+			// So we reset that to ensure fieldsets behave more like a standard block element.
+			// See https://github.com/twbs/bootstrap/issues/12359
+			// and https://html.spec.whatwg.org/multipage/#the-fieldset-and-legend-elements
+			minWidth: 0,
+			// Reset the default outline behavior of fieldsets so they don't affect page layout.
+			padding: 0,
+			margin: 0,
+			border: 0,
+		},
+
+		// 1. Correct the text wrapping in Edge and IE.
+		// 2. Correct the color inheritance from `fieldset` elements in IE.
+		legend: {
+			display: 'block',
+			width: '100%',
+			maxWidth: '100%', // 1
+			padding: 0,
+			marginBottom: '.5rem',
+			// @include font-size(1.5rem),
+			fontSize: 'inherit',
+			lineHeight: 'inherit',
+			color: 'inherit', // 2
+			whiteSpace: 'normal', // 1
+		},
+
+		//
+		// Correct element displays
+		//
+
+		template: {
+			display: 'none', // Add the correct display in IE
+		},
+
+		// Always hide an element with the `hidden` HTML attribute (from PureCSS).
+		// Needed for proper display in IE 10-.
+		'[hidden]': {
+			display: 'none !important',
+		},
+	};
+
+	// Typography styling
+	const styleTypography = {
+		// Document
+		html: {
+			fontSize: typography.fontSize, //10px
+		},
+
+		// Global type styling
+		body: {
+			fontFamily: typography.body.fontFamily,
+			fontWeight: typography.body.fontWeight,
+			fontSize: typography.body.fontSize, //1.4rem (14px)
+			lineHeight: typography.body.lineHeight, //1.428571429
+			color: typography.body.color,
+			fontFeatureSettings: '"liga" 1', // Enable OpenType ligatures in IE
+		},
+
+		// Globally reset all focus styling, only apply if keyboard user
+		':focus': {
+			...(isKeyboardUser ? typography.link.focus : { outline: 0 }),
+		},
+
+		// Class access to our font families
+		'.body-font': {
+			fontFamily: typography.body.fontFamily,
+		},
+		'.brand-font': {
+			fontFamily: typography.brand.fontFamily,
+		},
+
+		// Headings
+		'h1, h2, h3, h4, h5, h6': {
+			...typography.headings,
+		},
+
+		// Paragraphs
+		p: {
+			...typography.p,
+		},
+
+		// Definition lists
+		dl: {
+			...typography.dl,
+		},
+		dt: {
+			...typography.dt,
+		},
+		dd: {
+			...typography.dd,
+		},
+
+		// Abbreviation
+		'abbr[title]': {
+			...typography.abbr,
+		},
+
+		address: {
+			...typography.address,
+		},
+
+		blockquote: {
+			...typography.blockquote,
+		},
+
+		// Mark
+		mark: {
+			...typography.mark,
+		},
+
+		// Text selection
+		'::selection': {
+			...typography.selection,
+		},
+
+		// Links
+		a: {
+			...typography.link.default,
+
+			':hover': {
+				...typography.link.hover,
+			},
+		},
+	};
+
+	// Font @fontface styling
+	const styleFont = font;
```

Comment:
unnecessary, just use the font variable as is

Reply:
ok, will remove 'style' prefix from all emotion passed css object going forward

Reply:
I'm fine with the rename in the destructure